src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/Actions/InputNodeGraphActionHandlerBase.cs

```diff
@@ -70,7 +70,7 @@ public sealed override bool TryHandleRequest(IGraphContext graphContext)
                 }
 
                 using var scope = new GraphTransactionScope();
-                ProcessInputNode(graphContext, inputGraphNode, dependency, snapshot, viewProvider, projectPath, ref trackChanges);
+                ProcessInputNode(graphContext, inputGraphNode, dependency, snapshot, viewProvider, projectPath!, ref trackChanges);
```

Comment:
Let's wait and see if this is a compiler bug or a new restriction on the feature under .NET Framework.

Reply:
Oh wait, looking again I see you haven't updated the compiler in this PR. Are these warnings on `master` already?

Reply:
Yes.